src/riak_pb_dt_codec.erl

```diff
@@ -429,8 +447,10 @@ encode_operation(Op, map) ->
 -spec operation_type(#dtop{}) -> toplevel_type().
 operation_type(#dtop{counter_op=#counterop{}}) ->
     counter;
-operation_type(#dtop{set_op=#setop{}}) ->
+operation_type(#dtop{set_op=#setop{is_gset=false}}) ->
```

Comment:
I will have to double-check that `erlang_protobuffs` decodes booleans into atoms. It may decode into `0` and `1`.